src/components/bulk-email-tool/utils.js

```diff
@@ -0,0 +1,14 @@
+export const RECIPIENTS_DISPLAY_NAMES = {
+  myself: 'Myself',
+  staff: 'Staff and instructors',
+  learners: 'All students',
```

Comment:
I believe we generally prefer the term "Learner" to "Student" so this seems like a step backwards ?